### PR TITLE
Improve doc on cased dictionary hash lookup

### PR DESCRIPTION
**Pull request by jobergum**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by buinauskas — APPROVED** [review]
🙏🏻

File: en/performance/feature-tuning.html [hunk 1]
```diff
@@ -327,19 +327,40 @@ <h2 id="multi-lookup-set-filtering">Multi Lookup - Set filtering</h2>
...
+}
+</pre>
 <p>
-  E.g. if having 10M unique user_ids in the dictionary, a search for 1000 users per query,
-  <em>btree</em> dictionary would be 1000 lookup times log(10M),
-  while <em>hash</em> based would be 1000 lookups times 1.
+ If having 10M unique user_ids in the dictionary, a search for 1000 users per query,
+the <em>btree</em> dictionary would be 1000 lookup times log(10M),
+while <em>hash</em> based would be 1000 lookups times O(1). Still, the <em>btree</em> dictionary
+offers more flexiblity in terms of <a href="../reference/schema-reference.html#match">match</a> settings.
```
Comment (kkraune) [inline en/performance/feature-tuning.html:349]:
```suggestion
offers more flexiblity in terms of <a href="/en/reference/schema-reference.html#match">match</a> settings.
```

**Review by kkraune — APPROVED** [review]
Great!

**Review by baldersheim — COMMENTED** [review]
👍

**Pull request merged**